src/android/build-extras.gradle

```diff
@@ -1,5 +1,15 @@
+// Minimum SDK Version
+if (!project.hasProperty('cdvMinSdkVersion') || cdvMinSdkVersion < 15) {
+    ext.cdvMinSdkVersion = 15;
+}
+// For Push SDK, SDK version must be at least 23
+if (!project.hasProperty('cdvCompileSdkVersion') || cdvCompileSdkVersion < 23) {
```

Comment:
@dgonz7 We talked about the issue I was having with the Push SDK and Cordova. Since Push has a dependency on appcompat v7:23, I was having trouble compiling apps when including it. We figured out that we needed to set the API level to at least 23 (seemed to be a theme-related issue with material design) since Cordova defaults to API 22. One way of fixing this issue would be through the done in this gradle file Another way would be to manually change the project.properties files in platforms/android/ and platforms/android/CordovaLib/